WordPressCom-Stats-iOS/UI/WPStatsViewController.m

```diff
@@ -24,6 +26,14 @@ @interface WPStatsViewController () <StatsProgressViewDelegate, WPStatsSummaryTy
 
 @implementation WPStatsViewController
 
+- (nonnull instancetype)initWithStatsServiceCache:(nullable WPStatsServiceCache *)cache
```

Comment:
I think we can leave the service cache out of this layer since you can't change sites once you're in the stats view controller. 

There used to be a state issue with sharing the service instance between the different view controllers but I couldn't reproduce the issue. I think it had to do with the cached data in StatsEphemory being wiped out by forced refreshes on different screens.